honeybee_grasshopper_energy/src/HB Search Constructions.py

```diff
@@ -0,0 +1,65 @@
+# Honeybee: A Plugin for Environmental Analysis (GPL)
+# This file is part of Honeybee.
+#
+# Copyright (c) 2019, Ladybug Tools.
+# You should have received a copy of the GNU General Public License
+# along with Honeybee; If not, see <http://www.gnu.org/licenses/>.
+# 
+# @license GPL-3.0+ <http://spdx.org/licenses/GPL-3.0+>
+
+"""
+Search for available Constructions within the honeybee energy standards library.
+-
+
+    Args:
+        keywords_: Optional keywords to be used to narrow down the output list of
+            constructions. If nothing is input here, all available constructions
+            will be output.
+        parse_: If True, this component will automatically parse any strings of
```

Comment:
what if we call this input `split` or `multi` for multiple keywords?

I read `parse` as a verb to start parsing the data to match the keywords.

Reply:
Fair point. I was trying too hard to keep the component inputs short but I think I was overly ambitious.  What if we used `split_words_`?  That's still pretty short and I think it's the clearest name for the input that I could come up with.

Reply:
Actually, thinking about it more, I think `join_words_` is even clearer and I'll just invert the boolean.

Reply:
I like `join` which will be read as join keywords.